features/step_definitions/README.md

```diff
@@ -17,6 +17,6 @@ In particular:
   * Imperative messages
   * This is how we represent "intent"
   * Access via external adapter (if connected)
-* `Then` steps can only consult *projections* (read models)
```

Comment:
Projectors contain the logic to produce projections. Typical projections can be a read-model, but a projection can also be "sending an email".

In this sentence, I think it should still say *projections*